lib/matplotlib/lines.py

```diff
@@ -470,8 +470,7 @@ def contains(self, mouseevent):
         # application has set the error flags such that an exception is raised
         # on overflow, we temporarily set the appropriate error flags here and
         # set them back when we are finished.
-        olderrflags = np.seterr(all='ignore')
-        try:
+        with np.errstate(all='ignore'):
```

Comment:
This is a pre-1.6 addition to numpy correct?  (we are testing 1.6 with py2.7 on travis so this should be fine, but just being mildly paranoid)